File: proj8/proj_8_testers/c_files/test2.c
// Test more compex functions.
#include <stdio.h>

void foo() {
  int i = 0;
  while (i < 5) {
    if (i % 2 == 0) {
      printf("%d ", i);
    }
    i = i+ 1;

  }
  printf("\n");
}

int main() {
  foo();
  printf("Done\n");
  foo();
  printf("Done\n");
  return 0;
}

File: proj8/proj_8_testers/tests/test2.ml
# Tests more complex functions

begin
    foo() begin
        int i = 0;
        while i < 5 begin
            if i % 2 == 0 then begin
                write(i);
            end
            i := i + 1;
        end
        write("\n");
    end

    foo();
    write("Done\n");
    foo();
    write("Done\n");
end
